plugin/src/main/java/org/wso2/carbon/uuf/maven/bean/AppConfig.java

```diff
@@ -358,7 +358,7 @@ public void setSubmenus(List<MenuItem> submenus) {
 
         private List<String> csrfIgnoreUris = Collections.emptyList();
         private List<String> xssIgnoreUris = Collections.emptyList();
-        private Map<String, String> responseHeaders = Collections.emptyMap();
+        private ResponseHeaders responseHeaders;
```

Comment:
`private ResponseHeaders responseHeaders = new ResponseHeaders()`
So that if the webapp dev hasn't configured response header, the he/she will not get a NPE.